roles/glance/tasks/main.yml

```diff
@@ -41,7 +41,8 @@
     - glance-registry
 
 - name: glance config
-  template: src={{ item }} dest=/etc/glance mode=0640 owner=glance group=glance
+  template: src={{ item }} dest=/etc/glance mode={{ 0644 if 'policy' in item else 0640 }}
+            owner=glance group=glance
```

Comment:
In one you search for `policy.json` and in the other you search for `policy`, is there a reason for the divergence?